README.md

```diff
@@ -1,38 +1,81 @@
 # DeepSlide: A Sliding Window Framework for Classification of High Resolution Microscopy Images (Whole-Slide Images)
 
-This repository is a sliding window framework for classification of high resolution whole-slide images, often called microscopy or histopathology images. This is also the code for the paper [Pathologist-level Classification of Histologic Patterns on Resected Lung Adenocarcinoma Slides with Deep Neural Networks](https://www.nature.com/articles/s41598-019-40041-7). For a practical guide and implementation tips, see the medium post [Classification of Histopathology Images with Deep Learning: A Practical Guide](https://medium.com/health-data-science/classification-of-histopathology-images-with-deep-learning-a-practical-guide-2e3ffd6d59c5). 
+This repository is a sliding window framework for classification of high resolution whole-slide images, often called microscopy or histopathology images. 
+This is also the code for the paper 
+[Pathologist-level Classification of Histologic Patterns on Resected Lung Adenocarcinoma Slides with Deep Neural Networks](https://www.nature.com/articles/s41598-019-40041-7).
+ For a practical guide and implementation tips, see the Medium post 
+ [Classification of Histopathology Images with Deep Learning: A Practical Guide](https://medium.com/health-data-science/classification-of-histopathology-images-with-deep-learning-a-practical-guide-2e3ffd6d59c5). 
 
 
 ![alt text](figures/figure-2-color.jpeg)
 
-## Dependencies
-- [Python 3.6](https://www.anaconda.com/download/#macos)
-- [PyTorch](https://pytorch.org/)
-- [PIL](https://pillow.readthedocs.io/en/5.3.x/)
+## Changes in this Release
+- Use Python [multiprocessing](https://docs.python.org/3.6/library/multiprocessing.html)
+to speed up patch creation.
+- Replace [os](https://docs.python.org/3.6/library/os.html) with
+[pathlib](https://docs.python.org/3.6/library/pathlib.html) module.
+- Added [argparse](https://docs.python.org/3.6/library/argparse.html) support.
+- Rewrote patch processing code using vector operations.
+- Fully documented all functions and methods with docstrings and type hints.
+- Compute image means and standard deviations for normalization online.
+- Increased efficiency of code by rewriting inefficient code.
+
+## Known Issues and Limitations
+- Patch creation code doesn't find any patches in images that are slightly larger
+than the specified patch size.
+- Only 1 GPU supported.
+- Should work, but not tested on Windows.
+- In cases where no crops are found for an image, empty directories are created.
+Current workaround uses `try` and `except` statements to catch errors.
+- Image reading code expects colors to be in the RGB space. Current workaround
+is to keep first 3 channels.
```

Comment:
Why's this on a new line

Reply:
@jasonwei20 This is a formatting mistake.  I'll fix and commit.